### PR TITLE
[fr] add french translation for glossary/continuous_media (id #153), intrinsic_size (id #311)

### PR DESCRIPTION
**Pull request by ManuSquall**
add index translated file for [glossary/continuous_media/index.html](https://tristantheb.github.io/history-content/#153)

### CODE REVIEW

File: files/fr/glossary/continuous_media/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Médias Continus
```
Comment (audrasjb) [inline files/fr/glossary/continuous_media/index.html:2]: In French don't capitalize each word of a title like in English. Capitalize only the first word (and proper nouns) :)

File: files/fr/glossary/continuous_media/index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+slug: Glossary/Continuous_Media
+---
```
Comment (audrasjb) [inline files/fr/glossary/continuous_media/index.html:4]: I think we need a `translation_of: 'Glossary/Continuous_Media'` header here :)

**Review by audrasjb — CHANGES_REQUESTED** [review]

File: files/fr/glossary/continuous_media/index.html (hunk 2 above)
Comment (ManuSquall) [reply files/fr/glossary/continuous_media/index.html:5]: Thanks @audrasjb , do we have to always put this header within ``` ' ' ```? To correct it in my [previous PR](https://github.com/mdn/translated-content/pull/2056/files)

Comment (SphinxKnight) [reply files/fr/glossary/continuous_media/index.html:5]: Hello :) quotes are not mandatory when the string does not contain such string delimiters. (They don't hurt though)

File: files/fr/glossary/intrinsic_size/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Taille intrinsèquee
```
Comment (SphinxKnight) [inline files/fr/glossary/intrinsic_size/index.html:2]: minor typo with a superfluous e

File: files/fr/glossary/intrinsic_size/index.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+<p>La taille de <code>max-content</code> est l'inverse — dans le cas du texte, le texte s'afficherait aussi large que possible, sans enveloppement souple, même si un débordement était provoqué. La valeur du mot clé <code>max-content</code> expose ce comportement.</p>
+
+<p>Pour les images, la taille intrinsèque a la même signification — c'est la taille avec laquelle une image serait affichée si aucun CSS n'était appliqué pour modifier le rendu. Par défaut, les images sont supposées avoir une densité de "1x" pixels (1 pixel de périphérique = 1 pixel CSS) et donc la taille intrinsèque est simplement la hauteur et la largeur du pixel. La taille et la résolution intrinsèques de l'image peuvent être explicitement spécifiées dans les données {{Glossary("EXIF")}}. La densité de pixels intrinsèque peut également être définie pour les images à l'aide de l'attribut {{htmlattrxref("srcset", "img")}} (notez que si les deux mécanismes sont utilisés, la valeur <code>srcset</code> est appliquée "par dessus" la valeur EXIF).</p>
```
Comment (SphinxKnight) [inline files/fr/glossary/intrinsic_size/index.html:15]: Grammalecte says "par-dessus" and not "par dessus" :shrug:

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Minor stuff on the second page :)

File: files/fr/glossary/intrinsic_size/index.html (hunk 3 above)
Comment (ManuSquall) [reply files/fr/glossary/intrinsic_size/index.html:2]: Thanks!

File: files/fr/glossary/intrinsic_size/index.html (hunk 4 above)
Comment (ManuSquall) [reply files/fr/glossary/intrinsic_size/index.html:15]: My bad 😅, thank you!

**Review by SphinxKnight — APPROVED** [review]
Thanks, let's get this merged.

**Comment by ManuSquall** [comment]
thanks for the review and corrections

**Pull request merged**